ts

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env -S deno run
+// deno-lint-ignore-file prefer-const
+let message = 'Hello, Deno!';
+console.log(message);
```

Comment:
small nit, the formatter doesn't catch the missing trailing new line because the file doesn't contain an extension it recognises, but we should probably put one here.

Reply:
Sorry, this is a stale file used during testing. I'll remove it.